fundingmanager.go

```diff
@@ -146,7 +155,8 @@ type fundingManager struct {
 
 // newFundingManager creates and initializes a new instance of the
 // fundingManager.
-func newFundingManager(w *lnwallet.LightningWallet, b *breachArbiter) *fundingManager {
+func newFundingManager(w *lnwallet.LightningWallet, b *breachArbiter,
```

Comment:
To avoid passing in the `breachArbiter`, we can pass in only _exactly_ what the `fundingManager` needs which is a type: 
```
chan<- *lnwallet.LightningChannel
```

This denotes a **send-only** channel from the PoV of the `fundingManager` and also improves type safety a bit. At the construction site, we'd only need to pass in a single channel (the one belonging to the `breachArbiter`. This change will also make adding the planned unittest to the `fundingManager` easier as we won't need to create an entire instance of the `breachArbiter` to do so.